packages/InteractiveIcon/shared/animations.js

```diff
@@ -5,11 +5,12 @@ const animations = {
     },
   },
   scale: {
-    transition: 'transform 150ms ease-in-out',
```

Comment:
Out of curiosity, was this rule not be applied correctly and had to be moved to the next two selectors?

Reply:
Yes, that's correct.